eZ/Publish/API/Repository/ContentTypeService.php

```diff
@@ -89,7 +89,7 @@ public function updateContentTypeGroup(ContentTypeGroup $contentTypeGroup, Conte
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to delete a content type group
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException If  a to be deleted content type has instances
      *
-     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup $ContentTypeGroup
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup $contentTypeGroup
```

Comment:
Typos like this could probably go to `6.7` or `7.5` at least.

Reply:
Yeah, I can open the similar PR on 6.7.